### DISCUSSION TITLE
eze plugin keeps stopping working

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#803] [Q&A]
I use the win10 version of lanrargi, and the eze plugin always stops working after importing dozens of manga, so that I have to keep restarting shinobu when importing a lot of manga.

**Comment by Difegue** [comment]
👋 I think this is caused by the same problem you had in https://github.com/Difegue/LANraragi/issues/638 -- The file watcher is still iffy on Windows, alas.

What folder are you storing your manga in -- Is it a network drive?  
Sadly filesystem performance is usually hit-or-miss with netdrives and that's an issue I can't easily fix.

**Reply by nyanpasunyan** [reply]
Thank you for your reply. All my files are stored on the local hard drive.
